web/src/compositions/useNotifications.ts

```diff
@@ -1,5 +1,20 @@
-import Notifications, { notify } from '@kyvg/vue3-notification';
+import Notifications, { NotificationsOptions, notify } from '@kyvg/vue3-notification';
 
 export const notifications = Notifications;
 
-export default () => ({ notify });
+function notifyErr(err: unknown, args: NotificationsOptions | string): void {
+  // eslint-disable-next-line no-console
+  console.error(err);
+
+  let a = args;
+  if (typeof args === 'string') {
+    a = {
+      title: args,
+      type: 'error',
+    };
+  }
+
+  notify(a);
+}
```

Comment:
```suggestion
function notifyError(err: Error, args?: NotificationsOptions | string): void {
  // eslint-disable-next-line no-console
  console.error(err);

  if (args === undefined || typeof args === 'string') {
    notify({
      title: args || err.message,
      type: 'error',
    });
    return;
  }

  notify(args);
}
```

Reply:
err can not be Error as catch could also have other type

Reply:
Ohh. Would the update suggestion work? 🤔

Reply:
This looks good 👍